packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx

```diff
@@ -207,23 +212,22 @@ const ConditionsWidget: React.FC<ConditionsWidgetProps> = ({
           ) : (
             <>
               <Controller
-                name="search"
+                name="conditionSearch"
                 control={control}
                 render={({ field: { onChange, value, onBlur } }) => (
                   <ResponsiveWrapper isTablet={isTablet}>
                     <Search
                       autoFocus
                       ref={searchInputRef}
                       size="md"
-                      id="conditionsSearch"
+                      id="conditionSearch"
                       labelText={t('enterCondition', 'Enter condition')}
                       placeholder={t('searchConditions', 'Search conditions')}
-                      className={errors?.search && styles.conditionsError}
+                      className={errors?.conditionSearch && styles.errorOutline}
                       onChange={(e) => {
                         onChange(e);
                         handleSearchTermChange(e);
                       }}
-                      renderIcon={errors?.search && <WarningFilled />}
```

Comment:
Why is this icon being removed?

Reply:
That's my fault. Initially, it's not correctly mounted. I thought it would be fixed later, but I forgot to fix that after improving the logic. Thanks for pointing it out.